sensorprocess/lidarsensorprocess.go

```diff
@@ -16,128 +16,71 @@ import (
 
 // StartLidar polls the lidar to get the next sensor reading and adds it to the cartofacade.
 // Stops when the context is Done.
-func (config *Config) StartLidar(ctx context.Context) bool {
```

Comment:
We don't have to return a `bool` anymore since `StartLidar` only runs in online mode. The `bool` was used to indicate the status of offline mode.